src/memtester_parsing.py

```diff
@@ -0,0 +1,189 @@
+# (c) Open Compute Project
+# (c) Google LLC
+# (c) Igor Semenov (goshik92@gmail.com)
```

Comment:
Done. We should be careful with moving code when reorganizing repos. Dylan's authorship was erased from some of his code, IIRC.

Reply:
> Dylan's authorship was erased from some of his code, IIRC.

i wouldnt necessarily say so, that kind of origin is present in the git history. And over time, can you say a particular file was written by one person or another?
Arguably, in case of distribution, where the git history isnt immediately available, we should always have a pointer to origin, and generally speaking the group having worked on said artifact.

I however do not consider myself an authority on this matter. Different people can interepret this differently. Above is my opinion

Reply:
Git history is good enough. A was referring to a case when a project was moved from one repo to another without keeping  the history. That confused me, because I did not know who the authors were and whom to ask questions about the code. That is why I added myself to the copyright section initially: if the history got erased, people would know whom to talk to. But as long as we maintain the history properly, we do not need to mention authors directly in the code.